db/migrate/20200512132026_add_priority_to_todo_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriorityToTodoItems < ActiveRecord::Migration[6.0]
+  def change
+    add_column :todo_items, :priority, :string
```

Comment:
String priority columns, Are you sure this is a good way to go?
Will you be able to order items on the basis of their priority?

Reply:
Changed it to integer and was able to sort it using,
	default_scope { order({completed_at: :asc}, {priority: :asc}, {created_at: :desc}) }
in the "app/model/todo_item.rb".
But the problem is I want to display it as string on page, like 1 => VeryHigh, 2=> High, ... , 5 => VeryLow. But couldn't figure it how?